cmd/kubeadm/app/phases/selfhosting/selfhosting_test.go

```diff
@@ -508,8 +511,8 @@ func TestBuildDaemonSet(t *testing.T) {
 				t.Fatalf("failed to marshal daemonset to YAML: %v", err)
 			}
 
-			if !bytes.Equal(dsBytes, rt.dsBytes) {
-				t.Errorf("failed TestBuildDaemonSet:\nexpected:\n%s\nsaw:\n%s", rt.dsBytes, dsBytes)
+			if diff := cmp.Diff(rt.dsBytes, dsBytes); diff != "" {
+				t.Errorf("failed TestBuildDaemonSet (-want,+got):\n%s", diff)
```

Comment:
i'd prefer if avoid the go-cmp dependency.
also this package is being removed in 1.21, so using bytes.Equal is fine for now.

Reply:
Reverted. Which package is getting removed? `go-difflib` or `go-cmp`?

Reply:
thanks for the update.

> Reverted. Which package is getting removed?

`/phases/selfhosting/`